pages/linux/module.md

```diff
@@ -26,3 +26,7 @@
 - Unload all loaded modules:
 
 `module purge`
+
+- Specify additional search directory:
+
+`module use {{search_directory}}`
```

Comment:
I didn't see what is a "search directory" in the document. This command is used to add user-created modules according to page 21 from https://lmod.readthedocs.io/_/downloads/en/latest/pdf/.
```suggestion
- Specify user-created modules:

`module use {{path/to/modulefiles}}`
```